rpc/jsonrpc/server/http_server.go

```diff
@@ -21,6 +21,14 @@ import (
 	cmttime "github.com/cometbft/cometbft/types/time"
 )
 
+type ErrInvalidRemoteAddress struct {
+	Addr string
+}
+
+func (e ErrInvalidRemoteAddress) Error() string {
+	return fmt.Sprintf("invalid listening address %s (use fully formed addresses, including the tcp:// or unix:// prefix)", e.Addr)
+}
+
```

Comment:
this could be moved to errors and also add `Unwrap()`

Reply:
no `Unwrap()` was needed because there was no `Source` in the custom error type, but removed this error from this file since the same error was declared in the `jsonrpc/errors/errors.go` file.